### PR TITLE
Add missing docstrings

### PR DESCRIPTION
**Pull request by Amxx**
Fixes L-02
Fixes L-05

#### PR Checklist

- [ ] Tests
- [ ] Documentation
- [ ] Changeset entry (run `npx changeset add`)

### CODE REVIEW

**Review by ernestognw — APPROVED** [review]

File: contracts/proxy/transparent/TransparentUpgradeableProxy.sol [hunk 1]
```diff
@@ -15,7 +15,13 @@ import {ProxyAdmin} from "./ProxyAdmin.sol";
  * include them in the ABI so this interface must be used to interact with it.
  */
 interface ITransparentUpgradeableProxy is IERC1967 {
-    function upgradeToAndCall(address, bytes calldata) external payable;
+    /**
+     * @dev Upgrade the implementation of the proxy to `newImplementation`, and subsequently execute the function call
+     * encoded in `data`.
+     *
+     * See {{UUPSUpgradeable-upgradeToAndCall}}
+     */
```
[the remark this answers is not in the document]
Comment (ernestognw) [reply contracts/proxy/transparent/TransparentUpgradeableProxy.sol]: Not a big deal, I'm leaving it open and merging

[the remark this answers is not in the document]
Comment (Amxx) [reply contracts/proxy/transparent/TransparentUpgradeableProxy.sol]: I tend to agree. Should we remove that natspec comment and replace it with a non natspec that says "see ERC1967Upgradeable" ?

[the remark this answers is not in the document]
Comment (ernestognw) [reply contracts/proxy/transparent/TransparentUpgradeableProxy.sol]: And leave it with triple dashes `e.g. ///`? I'd agree.

[the remark this answers is not in the document]
Comment (Amxx) [reply contracts/proxy/transparent/TransparentUpgradeableProxy.sol]: it would be double dash (not natspec)

**Pull request merged**